addons/web/static/src/views/list/list_controller.scss

```diff
@@ -105,6 +105,10 @@
             user-select: none;  // Prevent unwanted selection while sorting
         }
 
+        .o_list_record_selector {
```

Comment:
shouldn't this be in list renderer.scss?

Reply:
There's no such file